.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,75 @@
+name: CI for Ambianic UI
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install npm
+      run: |
+        npm version
+        npm ci
+      env:
+        CI: true
+
+    - name: Test Suite
```

Comment:
@vickywane why do we need this workflow? Its intended for PWA testing, not serverless functions.

Reply:
I'll remove it.